src/anupSJun20/MiddleCharacterofAllWords.java

```diff
@@ -0,0 +1,27 @@
+package anupSJun20;
+/*Program 2 : 
+Create a new method to print middle character of all words.
+
+Input = "Harsh Maulik";
+Output : r is middle character in Harsh
+              u is middle character in Maulik */
+
+public class MiddleCharacterofAllWords {
+
+	static void findMiddleCharacter(String str) {
+		String[] newStr = str.split(" ");
+		for(int index = 0;index<newStr.length;index++) {
+			String allWords = (newStr[index]);
+			int charLength;
+			if(allWords.length()%2 ==0)
```

Comment:
we can have a solution to do it without if condition.
sop(allwords.charAt(allwords.length()-1)/2)